src/app/components/home-page/home-page.component.ts

```diff
@@ -1,33 +1,27 @@
 import { Component, OnInit } from '@angular/core';
-import { ApiService } from "./../api.service";
-
+import { ApiService } from './../api.service';
 
 @Component({
   selector: 'app-home-page',
   templateUrl: './home-page.component.html',
-  styleUrls: ['./home-page.component.css']
+  styleUrls: ['./home-page.component.scss'],
 })
-
-
-
 export class HomePageComponent implements OnInit {
   model = {
     left: true,
     middle: false,
-    right: false
+    right: false,
   };
 
-  recipes:any[];
+  recipes: any[];
   constructor(private recipesService: ApiService) {
     this.recipes = [];
   }
 
   ngOnInit(): void {
-    this.recipesService.getAllRecipes().subscribe((recipes: any[]) =>
-    {
+    this.recipesService.getAllRecipes().subscribe((recipes: any[]) => {
```

Comment:
I think you can shorten it to this.recipes = await this.recipesService.getAllRecipes().toPromise(). Note that you won't be able to use `await` inside `ngOnInit` so you'd have to export this code to some separate function